src/lib/guesserStrategies/perLetterEliminationStrategy.ts

```diff
@@ -16,22 +16,37 @@
  * You should have received a copy of the GNU General Public License
  * along with this program, namely the "LICENSE" text file.  If not,
  * see <https://www.gnu.org/licenses/gpl-3.0.html>.
- *****/
+ */
 
 import { sumBy } from 'lodash';
-import { IStrategyScoreMethod, NextWordGuesserStrategyBase } from '../nextWordGuesserStrategy';
+import NextWordGuesserStrategyBase from '../nextWordGuesserStrategy';
 import WordList from '../wordList';
 import { LetterAtPositionInWord, LetterAtPositionInWordRule } from '../letterAtPosition';
 
-export default class PerLetterEliminationStrategy extends NextWordGuesserStrategyBase implements IStrategyScoreMethod {
+/**
+ * A guessing strategy which scores the candidate words based on the letter frequency at each position.
+ */
+export default class PerLetterEliminationStrategy extends NextWordGuesserStrategyBase {
     constructor(...params: ConstructorParameters<typeof NextWordGuesserStrategyBase>) {
         super(...params);
     }
 
+    /**
+     * Counts the number of total possible letters (with repetition) in the given word list.
+     *
+     * @param wordList - the word list from which to count possible letters.
+     * @returns The count of total possible letters in the given word list.
+     */
     static totalPossibleLettersInWordList(wordList: WordList): number {
         return sumBy(wordList.possibleLetters, 'length');
     }
 
+    /**
+     * Maps the given guess to an array of `LetterAtPositionInWordRule` with `required` set to `Misplaced`.
+     *
+     * @param guess - the candidate guess
+     * @returns An array of rules with each letter at its position in the given guess marked as `Misplaced`.
```

Comment:
Nit: Consistency of casing and wording.
```suggestion
     * @returns an array of rules with each letter at its position in `guess` marked as `Misplaced`.
```